unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,38 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    String isoString = null;
+    try {
+      isoString = new String(byteArr, "US-ASCII");
```

Comment:
tarekauel,

before upload this code, i've tested it with a test case   ñ -> Ñ
byte[]={0x32,0x34,0x31}